man/man7/fi_opx.7

```diff
@@ -163,6 +163,22 @@ inclusive.
 .PP
 Default setting is 64.
 .TP
+\f[I]FI_OPX_RELIABILITY_MAX_UNCONGESTED_PINGS\f[R]
+Integer.
+This setting controls how many PING requests the reliability/replay
+function will issue per iteration of FI_OPX_RELIABILITY_SERVICE_USEC_MAX 
+in situations with less contending, outgoing traffic from the HFI.
+.PP
+Default setting is 128. Range of valid values is 1-65535.
+.TP
+\f[I]FI_OPX_RELIABILITY_MAX_CONGESTED_PINGS\f[R]
+Integer.
+This setting controls how many PING requests the reliability/replay
+function will issue per iteration of FI_OPX_RELIABILITY_SERVICE_USEC_MAX 
+in situations with more contending, outgoing traffic from the HFI.
+.PP
+Default setting is 4. Range of valid values is 1-65535.
+.TP
```

Comment:
FYI, man pages are updated automatically from the MarkDown files and there is no need to include the change here.

Reply:
@j-xiong If you want, I'll push an updated PR without the changes to fi_opx.7. Or we can just leave it and it will be written over. Any preference?

Reply:
It's fine to leave it as it is.